nilai-api/src/nilai_api/app.py

```diff
@@ -17,8 +17,7 @@ async def lifespan(app: FastAPI):
 
     yield {"redis": client, "redis_rate_limit_command": rate_limit_command}
 
-
-host = "nilai.sandbox.nilogy.xyz"
+host = "nilai-a779.nillion.network"
```

Comment:
This enables you to dynamically select the host based on the provided domain name for each container.  This way, even if `nilai-a779.nillion.network` disappears, it would still work based on the specific instance.

```suggestion
from nilai_common.config import SETTINGS
host = SETTINGS.host
```